Classes/Controller/ConfigurationController.php

```diff
@@ -37,6 +38,12 @@ class ConfigurationController extends AbstractModuleController
      */
     protected $defaultViewObjectName = FusionView::class;
 
+    /**
+     * @Flow\Inject
+     * @var FlashMessageContainer
+     */
+    protected $flashMessageContainer;
```

Comment:
We don`t need to inject the container here. The controller inherits from the flow action controller and therefore we can use the controllerContext.

```
$this->controllerContext->getFlashMessageContainer()->getMessagesAndFlush()
```

Reply:
You are right! I messed up in updating the pull request, so added a new one. Sorry for the inconvenience!